imap_processing/tests/ialirt/unit/test_process_codicelo.py

```diff
@@ -0,0 +1,104 @@
+import numpy as np
+import pytest
+
+from imap_processing import imap_module_directory
+from imap_processing.cdf.utils import load_cdf
+from imap_processing.ialirt.l0.process_codicelo import (
+    append_cod_lo_data,
+    find_groups,
+    process_codicelo,
+)
+from imap_processing.utils import packet_file_to_datasets
+
+
+@pytest.fixture(scope="session")
+def xtce_codicelo_path():
+    """Returns the xtce directory."""
+    return (
+        imap_module_directory / "ialirt" / "packet_definitions" / "ialirt_codicelo.xml"
+    )
+
+
+@pytest.fixture(scope="session")
+def binary_packet_path():
+    """Returns the xtce directory."""
+    return (
+        imap_module_directory
+        / "tests"
+        / "ialirt"
+        / "test_data"
+        / "l0"
+        / "apid01152.tlm"
+    )
+
+
+@pytest.fixture(scope="session")
+def codicelo_test_data():
```

Comment:
```suggestion
def codicelo_validation_data():
```

Should we call this "validation" data instead? In my mind, "test data" is input L0 data, whereas "validation data" is the final data to compare values against.